src/Faker/ORM/Doctrine/Populator.php

```diff
@@ -68,13 +84,24 @@ public function execute($entityManager = null)
             throw new \InvalidArgumentException("No entity manager passed to Doctrine Populator.");
         }
 
+        $insertCount = 0;
```

Comment:
Perhaps rename to `$entityCount`?

The entities will only have been inserted once `$entityManager->flush()` has been called. As long as that has not happened, the variable name is a bit misleading.